### ISSUE TITLE
Feature Request:  Anthology ebooks email handling

### ISSUE THREAD

**Issue by Mieekaserra** [JimmXinu/FanFicFare#786]
I have my calibre setup to check for existing anthologies and reject if found.

Ideally - i'd like a email to update the anthology Ebook if it finds one, rather than ignore the mail - but thats probably too complex.

So instead - i'd like it to be able to tag the Anthology Ebook with something i can search for so i know which ones i need to manually process the updates.

In addition - if a update is found for a new story in a series but not currently in the anthology for that series to not create a ebook for the new update.

**Comment by JimmXinu** [comment]
Addressing your points in order:

1. FFF is designed to do multiple individual story downloads at once.  The anthology feature kind of hijacks that ability to group multiple individual story downloads as an anthology.  Consequently, FFF can't do both at the same time, which makes launching an anthology update from an email notification impossible in the current architecture--we have to assume there may be other individual story update emails.

2. It might be possible for FFF to use Calibre's Mark books feature to indicate existing anthology books that had story URLs that presented the 'Skip Anthology Story' dialog.
  Caveat: FFF can already be configured to use Marks at the end of a download session.  That would replace any anthology Marks, but only after you accepted the 'Proceed with updates' dialog.

3. A story is identified as part of an existing anthology book when it's `seriesUrl` metadata matches an existing book's `identifier:url`.  Whether it is already in the existing anthology or not shouldn't matter, it should be treated the same.

So, I will look at Mark'ing anthologies because I think it would be useful, but I'm afraid that's as good as it's likely to get right now.

**Comment by Mieekaserra** [comment]
Hi Jimm,
This is a example of point 3. I'm assuming its because it has multiple Anothologies - and your only matching the first?

https://archiveofourown.org/works/35266612/chapters/87887950

I have the anthology "like a malfoy" in my collection, but its created a book with the series "From Samhain to the Solstice 2021" instead of rejecting it

**Comment by JimmXinu** [comment]
Correct.  The 'official' series as far as FFF is concerned is [From Samhain to the Solstice 2021](https://archiveofourown.org/series/2593717).  [Like a Malfoy](https://archiveofourown.org/series/1797043) is in the site specific series01 set of metadata.

You could use `replace_metadata` settings to change the main series, but it would only help with updates, not new stories.  It would also affect the title page FFF generates.  Example:

```
[https://archiveofourown.org/works/35266612]
add_to_replace_metadata:
 seriesUrl=>^.*$=>https://archiveofourown.org/series/1797043
 series=>^.*$=>Like a Malfoy [5]
```
